### ISSUE TITLE
Variational Autoencoder Development

### ISSUE THREAD

**Issue by godonan** [godonan/Project-Apple#2]
- Develop a variational autencoder in Tensorflow 2.
- Test the autoencoder against MNIST datasets.
- Use generic ELBO as the loss function.
